docusaurus/docs/Android/04-compose/03-channel-components/02-channel-list-header.mdx

```diff
@@ -1,10 +1,10 @@
 # ChannelListHeader
 
-The `ChannelListHeader` component allows you to display a header for the channel screen. It sets up the following:
+The `ChannelListHeader` component allows you to display a header for the channels screen. It sets up the following:
 
-* `leadingContent`: Shows the current user image. The style is applied from the `ChatTheme` wrapper. <!-- TODO WIP PAGE -->
-* `centerContent`: A component that shows the title of the header, or a loading view if there's no network available.
-* `trailingContent`: A customizable trailing action shown at the end of the header, exposed as a parameter.  `DefaultChannelListHeaderAction` by default.
+* User avatar: Shows the current user image.
```

Comment:
I think we should bold the parts before the colon for visibility's sake.

E.g. - "**User avatar:** Shows the current user image"